File: year2020/month12/day1202/ms_1202.cpp
#include <iostream>
#include <string>

using namespace std;

string solution(string s);

int main() {
	string s;

	getline(cin, s);

	if (1 <= s.length() && s.length() <= 100) {
		cout << solution(s) << endl;
	}

	return 0;
}

string solution(string s) {
	string answer = "";
	int half = s.length() / 2;

	if (s.length() % 2)
		answer += s.substr(half, 1);
	else
		answer += s.substr(half - 1, 2);

	return answer;
}

File: year2020/month12/day1202/ms_1202.cs
using System;

namespace algorithm_113
{
    class ms_1202
    {
        static string solution(string s) 
        {
            string answer = "";
            int half = s.Length / 2;

            if(s.Length % 2 == 0)
                answer += s.Substring(half-1, 2);
            else
                answer += s.Substring(half, 1);

            return answer;
        }

        static void Main(string[] args)
        {
            string s = Console.ReadLine();

            if(1 <= s.Length && s.Length <= 100)
            {
                Console.WriteLine(solution(s));
            }
        }
    }
}
